pkg/kubectl/cmd/rollingupdate.go

```diff
@@ -172,10 +170,19 @@ func RunRollingUpdate(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, arg
 		if err != nil {
 			return err
 		}
+
+		var newName string
+		if len(args) >= 2 {
+			newName = args[1]
+		} else {
+			rename = true
```

Comment:
This variable name is confusing. If rename = true, the rc will keep its original name after the update. Consider calling this e.g. "keepOldName" to better convey intended use.